include/picongpu/plugins/radiation/Radiation.hpp

```diff
@@ -96,7 +92,7 @@ namespace picongpu
                  * The second dimension is used to store intermediate results if command
                  * line option numJobs is > 1.
                  */
-                GridBuffer<Amplitude, 2>* radiation;
+                GridBuffer<Amplitude<>, 2>* radiation;
```

Comment:
Perhaps there could be a `using` alias for `Amplitude<>`, so that it can be changed in one place if needed? Also, then the using code looks a little cleaner as it is decouple from knowledge that `Amplitude` is a template instantiated with detault arguments. But it's a matter of taste, not important.

Reply:
I completely agree. In a first attempt I messed up and tried to define a `using Amplitude = Amplitude<>` on a namespace level. As of 539449d  the `using` statement is in the functions and works properly.